app/halgo/Halgo/CLA/Command/Node.hs

```diff
@@ -122,6 +114,7 @@ cmdPrintBlock rnd url = withNode url $ \(_, api) -> do
       mapM_ putJson txs
     _ -> putTextLn "No block found"
 
+{-# DEPRECATED cmdNodeFetchAccount "Use `cmdIndexerFetchAccount` instead" #-}
```

Comment:
I find it strange to have `DEPRECATED` pragma in **executable**. The only possible user of this function is this executable, it's not exported externally. Are we using this function in this executable? If no, we can remove it, there is no user and can't be any.